changelog/fragments/add_common_kb.yaml

```diff
@@ -0,0 +1,51 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (ansible/v1,helm/v1) Add componentConfig option. For further information check [here](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html)
```

Comment:
My only concern here is; Have any reason for we do not add this option for Ansible/Helm as well? 
c/c @jmrodri @estroz